roles/openshift_cfme/tasks/main.yml

```diff
@@ -0,0 +1,178 @@
+---
+# PASSED IN ROLE VARIABLES:
+#
+# - template_dir - A remote temp dir to template the PVs in
+#
+# TODO: Refactor creation of this temp dir to a JIT approach to
+# improve idempotency.
+######################################################################
+
+######################################################################
+# Users, projects, and privileges
+
+- name: Ensure the CFME user exists
+  oc_user:
+    state: present
+    username: "{{ openshift_cfme_user }}"
+
+- name: Ensure the CFME namespace exists with CFME user as admin
+  oc_project:
+    state: present
+    name: "{{ openshift_cfme_project }}"
+    display_name: "{{ openshift_cfme_project_description }}"
+    admin: "{{ openshift_cfme_user }}"
+
+- name: Ensure the CFME namespace service account is privileged
+  oc_adm_policy_user:
+    namespace: "{{ openshift_cfme_project }}"
+    user: "{{ openshift_cfme_service_account }}"
+    resource_kind: scc
+    resource_name: privileged
+    state: present
+
+######################################################################
+# Service settings
+
+- name: Ensure bulk image import limit is tuned
+  yedit:
+    src: /etc/origin/master/master-config.yaml
+    key: 'imagePolicyConfig#maxImagesBulkImportedPerRepository'
```

Comment:
Should we also set this in the master config template so that byo doesn't flip/flop this? Not sure if we'd want to configure for everyone, just wondering..